Autopilot/Inc/IMUThreadInterface.h

```diff
@@ -0,0 +1,16 @@
+#ifndef IMU_THREAD_INTERFACE_H
+#define IMU_THREAD_INTERFACE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+void IMUThreadInterfaceInit(void);
+void IMUThreadInterfaceExecute(void);
+
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif 
```

Comment:
newline. Last time i am writing it. Please go through the "Files changed" section of this PR and find the other places where this is